wallet/urls.py

```diff
@@ -5,8 +5,8 @@
 
 
 router = routers.DefaultRouter()
-router.register(r'category', CategoryViewSet)
-router.register(r'expense', ExpenseViewSet)
+router.register(r'category', CategoryViewSet, basename='category')
```

Comment:
Usually the list api should be in plural form, like `categories`

Reply:
But this is not just a list API. This is a modelViewSet, the add, edit, delete will use the same basename. Actions are distinguished by form submit method.